changelogs/fragments/347-ansible-test-python-requires.yml

```
@@ -0,0 +1,2 @@
+trivial:
+  - ci - define `python_requires` for `ansible-test` (https://github.com/ansible-collections/community.digitalocean/issues/347).
```

This is RST, so monospace/code text uses double backticks:

```suggestion
  - ci - define ``python_requires`` for ``ansible-test`` (https://github.com/ansible-collections/community.digitalocean/issues/347).
```